opwen_email_server/celery/celery.py

```diff
@@ -0,0 +1,16 @@
+from __future__ import absolute_import
```

Comment:
Given that we're in Python 3 only, this import shouldn't be required.

Reply:
👍🏽